.changes/v2.24.0/660-improvements.md

```diff
@@ -0,0 +1,3 @@
+* `types.ExternalNetworkV2` adds new fields `NatAndFirewallServiceIntention` and
```

Comment:
Nitpick, just to use same structure as previous releases:
```suggestion
* Added new fields `NatAndFirewallServiceIntention` and `NetworkRouteAdvertisementIntention` to `types.ExternalNetworkV2`, which......
```

Reply:
Fixed.